.clang-tidy

```
@@ -54,6 +54,20 @@ CheckOptions:
     value:           CamelCase
   - key:             readability-identifier-naming.FunctionIgnoredRegexp
     value:           'name|message|make_error_code'
+  - key:             readability-identifier-naming.ClassCase
+    value:           CamelCase
+  - key:             readability-identifier-naming.ClassIgnoredRegexp
+    value:           'is_error_condition_enum'
+  - key:             readability-identifier-naming.MemberCase
+    value:           lower_case
+  - key:             readability-identifier-naming.MemberSuffix
+    value:           _
+  - key:             readability-identifier-naming.MemberIgnoredRegexp
+    value:           '^._$|^k0_._$|^kinf_._$|^Fc_$|^niLU_.*$|[LU]ij_.*$'
```

Rethinking this. Maybe we shouldn't require certain case, so we don't need to list exception cases for math parameters. @mattldawson, @K20shores, @mwaxmonsky, what do you think? 
```suggestion
  - key:             readability-identifier-naming.MemberCase
    value:           aNy_CasE
  - key:             readability-identifier-naming.MemberSuffix
    value:           _
```

---

That seems reasonable; we do work in a math field after all and not every variable can easily be named with camel case

---

I agree with you. I updated it to allow any case for variable names